src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -822,14 +821,13 @@ static void create_rr_locked(grpc_exec_ctx* exec_ctx, glb_lb_policy* glb_policy,
   pending_ping* pping;
   while ((pping = glb_policy->pending_pings)) {
     glb_policy->pending_pings = pping->next;
-    GRPC_LB_POLICY_REF(glb_policy->rr_policy, "rr_handover_pending_ping");
```

Comment:
Removing this is wrong. The only changes you should make to grpclb.cc would be to add the `on_notify` and `on_ack` closures to `wrapped_notify_arg`, instead of removing it.

Reply:
Added back.

Reply:
I don't see it, but based on your other comment, it's probably fine.

Reply:
Looks like it was hidden by github. The added `GRPC_LB_POLICY_REF`s were on line 840 and line 845 in 625a5c05456a3001ecd4519b133aecd61e5b333b.